Testing/integration/VM/bash_control.sh

```diff
@@ -3,5 +3,5 @@
 # through applescript.
 # It's run as part of the template VM creation process.
 
-osascript -e 'tell application "System Preferences" to activate'
-osascript -e 'tell application "System Events" to tell process "System Preferences" to click menu item "Profiles" of menu 1 of menu bar item "View" of menu bar 1'
+osascript -e 'tell application "System Settings" to activate'
```

Comment:
I think `System Preferences` should be an env var so we're not editing this everywhere.

Reply:
This was a nit.